endpoints/openrtb2/sample-requests/custom-currency/errors/non-usd-cur-custom-rates-usepbs-false.json

```diff
@@ -0,0 +1,53 @@
+{
+  "description": "BidRequest defines its own custom rates in the root.ext and sets the usepbsrates to false, not allowing for PBS to use its default converter and defaulting to a price of 0",
+  "config": {
+    "currencyRates":{
+      "USD": {
+        "MXN": 5.09
+      }
+    },
+    "mockBidder": {
```

Comment:
This pull request also implements the ability to define a mock bid within the JSON test